cmake/projects/rabit/hunter.cmake

```diff
@@ -14,11 +14,11 @@ hunter_add_version(
     PACKAGE_NAME
     rabit
     VERSION
-    0.0.0-p0
```

Comment:
Please keep old version.

Reply:
Ok.  I thought it would be better to remove broken releases, but I can see the problems this could cause

Reply:
Done.